crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -59,16 +113,41 @@ const transform = (
       {
         resource,
         context: {
+          currentTraceSpan: new DummySpan(),
           rootContext: contextDir,
           getOptions() {
             const entry = this.loaders[this.loaderIndex];
             return entry.options && typeof entry.options === "object"
               ? entry.options
               : {};
           },
+          getResolve: () => ({
+            // [TODO] this is incomplete
```

Comment:
We should have a `__turbopack_require_context__.resolve()` which is our module runtime's resolver.